tests/insert-pragma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,87 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`module-with-late-docblock.js 1`] = `
+function foo(bar) { 
+  return bar + 3 + 4;
+}
+
+/**
+ * Some notes that should not be appended to
+ */ 
+const fruit = 'tomatoes';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/** @format */
+function foo(bar) {
+  return bar + 3 + 4;
+}
+
+/**
+ * Some notes that should not be appended to
+ */
+
+const fruit = "tomatoes";
+
+`;
+
+exports[`module-with-pragma.js 1`] = `
+/**
+ * Here lie words above pragmas
+ * 
+ * @flow
+ * @format
+ */
+
+function foo(bar) {
+  return bar + 3 + 4;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * Here lie words above pragmas
+ * 
+ * @flow
+ * @format
+ */
+
+function foo(bar) {
+  return bar + 3 + 4;
+}
+
+`;
+
+exports[`module-without-docblock.js 1`] = `
+function foo(bar) { 
+  return bar + 3 + 4;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/** @format */
+function foo(bar) {
+  return bar + 3 + 4;
+}
+
+`;
+
+exports[`module-without-pragma.js 1`] = `
+/**
+ * @flow
+ * 
+ * Words below the wrong pragma
+ */
+
+function foo(bar) { 
+  return bar + 3 + 4;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * 
+ * 
+ * Words below the wrong pragma
```

Comment:
Sorry for the confusion, but could you change this to to trim leading newline characters (`/^(\r?\n)+/`) from the comments. `trim()` is too much.

Reply:
Thats what I suggested here: https://github.com/prettier/prettier/pull/2865#discussion_r142273376 

:).   Will do

Reply:
Do you think the fix belongs in `jest-docblock` instead?